specification/dartLangSpec.tex

```diff
@@ -2528,29 +2528,19 @@ \section{Classes}
 A concrete class must fully implement its interface:
 \BlindDefineSymbol{C, I, m}%
 Let $C$ be a concrete class with interface $I$.
-Assume that $I$ has an accessible member signature $m$.
+Assume that $I$ has an accessible member signature $m$,
+and $C$ does not declare a member with the same name as $m$.
```

Comment:
Reasonable constraint since a member declared in the class will force the class interface member signature to match.

Reply:
Right, this is basically just removing some redundancy: The rule in 'Instance Methods' takes care of declarations in `$C$`, this rule takes care of declarations inherited by `$C$` (when that's a concrete class). Without line 2532, this rule covers both (there is no contradiction, but the redundancy is confusing).